webapp/src/components/system_console/no_bots_page.tsx

```diff
@@ -0,0 +1,55 @@
+import {PlusIcon} from '@mattermost/compass-icons/components';
+import React from 'react';
+import styled from 'styled-components';
+
+import {PrimaryButton} from 'src/components/assets/buttons';
+import SparklesGraphic from 'src/components/assets/sparkles_graphic';
+
+import {PanelContainer} from './panel';
+
+type Props = {
+    onAddBotPressed: () => void;
+};
+
+const NoBotsPage = (props: Props) => {
+    return (
+        <StyledPanelContainer>
+            <SparklesGraphic/>
+            <Title>{'No AI bots added yet'}</Title>
+            <Subtitle>{'To get started with AI copilot, add an AI bot'}</Subtitle>
+            <PrimaryButton onClick={props.onAddBotPressed}>
+                <StyledPlusIcon/>
+                {'Add an AI Bot'}
+            </PrimaryButton>
+        </StyledPanelContainer>
+    );
+};
+
+const StyledPlusIcon = styled(PlusIcon)`
+	margin-right: 8px;
+	width: 18px;
+	height: 18px;
+`;
+
+const StyledPanelContainer = styled(PanelContainer)`
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	gap: 16px;
+	padding-bottom: 32px;
```

Comment:
Assuming this is only used for the empty state, can we make it `padding-bottom: 56px` to center things a bit better in the empty state?

Reply:
This is the result of that change: 
![image](https://github.com/mattermost/mattermost-plugin-ai/assets/3191642/54018df0-5902-4f45-8f15-6e166940997a)

Reply:
Looks good. Thanks!